app/src/main/java/ch/epfl/sweng/SDP/home/LeaderboardActivity.java

```diff
@@ -347,7 +346,14 @@ private ValueEventListener initializeFriendsButton() {
                 @Override
                 public void onDataChange(@NonNull DataSnapshot dataSnapshot) {
                     if (dataSnapshot.exists()) {
-                        setImageResource(R.drawable.remove_friend);
+                        int status = dataSnapshot.getValue(int.class);
+                        if (status == SENT.ordinal()) {
+                            setBackgroundResource(R.drawable.pending_friend);
```

Comment:
Replace by setImageResource, otherwise the image looks stretched

Reply:
yep, sorry